packages/ui/theme-builder/scripts/deploy.mjs

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env zx
+/* eslint-disable */
+import 'zx/globals';
+
+process.env.FORCE_COLOR = '1';
+
+if ((await which('vercel', { nothrow: true })) === null) {
+  echo('Vercel CLI not found. Please install it by running `npm i -g vercel`');
+  process.exit(1);
+}
+
+// build top-level
+await within(async () => {
+  // repo root
+  cd('../../../');
+
+  // turbo build
+  await spinner('Building dependencies...', () => $`npx turbo build --filter=ui...`);
```

Comment:
The `spinner()` usage in this file is because these commands execute subcommands themselves, so the output isn't getting fully piped through to this script. The spinner gives _some_ feedback.